island-is/island.is
# feat(samradsgatt): Add pagination my reviews

# Add pagination to my reviews page

https://veflausnir.atlassian.net/browse/KAM-932

## What

Created a pagination component
Added the component to the front page and to my reviews page.

## Why

...

## Screenshots / Gifs

<img width="397" alt="image" src="https://user-images.githubusercontent.com/106164521/223973461-b533069b-e449-4851-ba43-3b5fdc802a50.png">

<img width="381" alt="image" src="https://user-images.githubusercontent.com/106164521/223973776-01f842e4-e71c-4be1-9a5f-2d8a2d6da805.png">


## How to test

- Checkout branch feat/KAM-932-pagination-my-review
- Run the project locally : yarn start consultation-portal
- Run the api locally: yarn start api
- Look at the main page. Should still have pagination.
- Navigate to http://localhost:4200/umsagnir. Should have pagination.
- Observe and test.

## Checklist:
